### DISCUSSION TITLE
How do you do a custom alias for cmi cheque?

### DISCUSSION THREAD

**Discussion by redalpha123123** [Zrips/CMI#5219] [Q&A]
I can not seem to figure out how to put [playerName] in my custom commands. It always says Usage: /cheque (playerName) [amount]. Like I didnt code it right.

I do add [playerName] to the custom alias and add hasmoney:[amount]! 

so i make the custom command like this.

/cheque
- cmi cheque [playerName] hasmoney:[amount]!

I get an error = Usage: /cheque (playerName) [amount]

**Comment by CloudeLecaw** [comment]
u want a custom alias for that?

```
  scheck:
    Cmds:
    - cmi cheque $1-
```

this will work 100% equal to cmi cheque

What you wanted to create a little script that only got executed if player has amount of money?
i can add this for you if needed

**Comment by CloudeLecaw** [comment]
```
  moneypaper:
    Cmds:
    - hasitem:paper-1@! You need to hold a paper in your hand your write a cheque.
    - hasitem:paper-1#! hasmoney:$1@! You dont have enought money to create this cheque.
    - hasitem:paper-1#! hasmoney:$1#! cmi cheque $1-
    - msg! [playerName] You created a cheque abount $1 money.

```
This would allow players to create cheques - not create them for others ;)
command useage **/moneypaper AMOUNT**

**Comment by ZEROOblivious** [comment]
I've done my own alias for it myself. 

``` withdraw:
    Cmds:
    - cmi cheque $1-

**Comment by CloudeLecaw** [comment]
thats what customalias.yml is for :D
